source/common/http/utility.cc

```diff
@@ -369,10 +370,17 @@ std::string Utility::makeSetCookieValue(const std::string& key, const std::strin
 }
 
 uint64_t Utility::getResponseStatus(const ResponseHeaderMap& headers) {
+  absl::StatusOr<uint64_t> response_status_or_absl_status = getResponseStatusOr(headers);
+  RELEASE_ASSERT(response_status_or_absl_status.ok(),
+                 ":status must be specified and a valid unsigned long");
```

Comment:
Use the status message in case it ever changes

Reply:
Done